plugin/pkg/shoot/seedmanager/admission_test.go

```diff
@@ -90,20 +91,26 @@ var _ = Describe("seedmanager", func() {
 					},
 				},
 			}
-		)
-
-		BeforeEach(func() {
-			admissionHandler, _ = New()
-			admissionHandler.AssignReadyFunc(func() bool { return true })
-			gardenInformerFactory = gardeninformers.NewSharedInformerFactory(nil, 0)
-			admissionHandler.SetInternalGardenInformerFactory(gardenInformerFactory)
 
-			seed = *seedBase.DeepCopy()
-			shoot = *shootBase.DeepCopy()
-		})
+			defaultAdmissionConfiguration = seedmanagerapi.AdmissionConfigurationSeedmanager{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "seedmanager.admission.gardener.cloud/v1alpha1",
```

Comment:
You mixed up Kind/APIVersion here :-)
Also, you're almost already using my suggestion for how to rename the API group ;-)